openmdao/docs/advanced_guide/implicit_comps/n2.html

```diff
@@ -2308,7 +2310,7 @@ <h1>OpenMDAO Partition Tree and N<sup>2</sup> diagram.</h1>
 
             text += "<br />self.connect(\"" + unknownName + "\", \"" + paramName + "\")";
         }
-        if(true) {
+        if(false) {
```

Comment:
Are these `if` blocks frequently turned on/off?  Otherwise you could just delete these, since they are effectively commented out.

Reply:
This file is created via a Python script that flips the true/false based on an option to the Python function. We could make that code more intelligent to remove the "commented out" block but it doesn't like it would be worth it.